programs/open_creator_protocol/src/state.rs

```diff
@@ -83,7 +83,6 @@ impl Policy {
                 if result.condition_result.status != Status::Met {
                     msg!("Policy does not match: {}", result.condition_result.name);
                     msg!("fact: {}", fact);
-                    msg!("json_rule: {}", json_rule);
```

Comment:
we can already find the policy rules from the tx accounts, so saving the logging here